src/DssSpell.sol

```diff
@@ -40,163 +40,170 @@ contract DssSpellAction is DssAction {
     //
 
     // --- Rates ---
-    uint256 constant ZERO_PCT_RATE            = 1000000000000000000000000000;
-    uint256 constant ONE_FIVE_PCT_RATE        = 1000000000472114805215157978;
+    uint256 constant ZERO_PCT_RATE           = 1000000000000000000000000000;
+    uint256 constant ZERO_ONE_PCT_RATE       = 1000000000031693947650284507;
+    uint256 constant ONE_PCT_RATE            = 1000000000315522921573372069;
+    uint256 constant ONE_FIVE_PCT_RATE       = 1000000000472114805215157978;
+    uint256 constant TWO_PCT_RATE            = 1000000000627937192491029810;
+    uint256 constant TWO_FIVE_PCT_RATE       = 1000000000782997609082909351;
+    uint256 constant TWO_SEVEN_FIVE_PCT_RATE = 1000000000860244400048238898;
+    uint256 constant THREE_PCT_RATE          = 1000000000937303470807876289;
+    uint256 constant FOUR_PCT_RATE           = 1000000001243680656318820312;
+    uint256 constant SIX_PCT_RATE            = 1000000001847694957439350562;
+    uint256 constant SIX_FIVE_PCT_RATE       = 1000000001996917783620820123;
 
     // --- Math ---
-    uint256 constant MILLION                  = 10 ** 6;
-    uint256 constant RAD                      = 10 ** 45;
-
-    // --- WBTC-C ---
-    address constant MCD_JOIN_WBTC_C          = 0x7f62f9592b823331E012D3c5DdF2A7714CfB9de2;
-    address constant MCD_CLIP_WBTC_C          = 0x39F29773Dcb94A32529d0612C6706C49622161D1;
-    address constant MCD_CLIP_CALC_WBTC_C     = 0x4fa2A328E7f69D023fE83454133c273bF5ACD435;
-
-    // --- PSM-GUSD-A ---
-    address constant MCD_JOIN_PSM_GUSD_A      = 0x79A0FA989fb7ADf1F8e80C93ee605Ebb94F7c6A5;
-    address constant MCD_CLIP_PSM_GUSD_A      = 0xf93CC3a50f450ED245e003BFecc8A6Ec1732b0b2;
-    address constant MCD_CLIP_CALC_PSM_GUSD_A = 0x7f67a68a0ED74Ea89A82eD9F243C159ed43a502a;
-    address constant MCD_PSM_GUSD_A           = 0x204659B2Fd2aD5723975c362Ce2230Fba11d3900;
-
-    // --- Wallets + Dates ---
-    address constant SAS_WALLET     = 0xb1f950a51516a697E103aaa69E152d839182f6Fe;
-    address constant IS_WALLET      = 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4;
-    address constant DECO_WALLET    = 0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7;
-    address constant RWF_WALLET     = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
-    address constant COM_WALLET     = 0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9;
-    address constant MKT_WALLET     = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
-
-    uint256 constant DEC_01_2021    = 1638316800;
-    uint256 constant DEC_31_2021    = 1640908800;
-    uint256 constant JAN_01_2022    = 1640995200;
-    uint256 constant APR_30_2022    = 1651276800;
-    uint256 constant JUN_30_2022    = 1656547200;
-    uint256 constant AUG_01_2022    = 1659312000;
-    uint256 constant NOV_30_2022    = 1669766400;
-    uint256 constant DEC_31_2022    = 1672444800;
-    uint256 constant SEP_01_2024    = 1725148800;
+    uint256 constant MILLION = 10 ** 6;
+    uint256 constant BILLION = 10 ** 3;
```

Comment:
this should be:
`uint256 constant BILLION = 10 ** 9;`

Reply:
Thanks for spotting that, fixed on commit e6c3bb79acd406220c0b45a6bb1db8f02cdde59f.